store_monitoring/main.py

```diff
@@ -14,23 +14,35 @@ def get_db():
     finally:
         db.close()
 
-@app.post("/insert-store-status/")
-def store_status(db: Session = Depends(get_db)):
-    crud.insert_store_status(db)
-    return {"message": "Store status data inserted successfully."}
+# @app.post("/insert-store-status/")
+# def store_status(db: Session = Depends(get_db)):
+#     crud.insert_store_status(db)
+#     return {"message": "Store status data inserted successfully."}
 
 
-# def menu_hour(db: Session = Depends(get_db)):
-#     crud.insert_menu_hours(db)
-#     return 
+# # def menu_hour(db: Session = Depends(get_db)):
+# #     crud.insert_menu_hours(db)
+# #     return 
 
-@app.post("/insert-bq_results/")
-def bq_results(db: Session = Depends(get_db)):
-    crud.insert_bq_results(db)
-    return {"message": "bq results data inserted successfully."}
+# @app.post("/insert-bq_results/")
+# def bq_results(db: Session = Depends(get_db)):
+#     crud.insert_bq_results(db)
+#     return {"message": "bq results data inserted successfully."}
 
 
-@app.post("/menu_hour")
-async def menu_hour(background_tasks: BackgroundTasks, db: Session = Depends(get_db)):
-    background_tasks.add_task(crud.insert_menu_hours, db)
-    return {"message": "Menu hours insertion request received and enqueued for background execution"}
+# @app.post("/menu_hour/")
+# async def menu_hour(background_tasks: BackgroundTasks, db: Session = Depends(get_db)):
+#     background_tasks.add_task(crud.insert_menu_hours, db)
+#     return {"message": "Menu hours insertion request received and enqueued for background execution"}
+
+# @app.get("/business_hours/")
+# def business_hours(db:Session=Depends(get_db)):
+#     return crud.unique_store_id(db)
+
+@app.post("/trigger_report/")
+async def trigger_report(background_tasks: BackgroundTasks, db:Session=Depends(get_db)):
+    background_tasks.add_task(crud.insert_store_status(db))
```

Comment:
Are you sure you want to put them into separate tasks?

If yes, how would you know when all of them are done and you need to start processing?

Reply:
I will see to it